criu/flog.c

```diff
@@ -150,66 +155,87 @@ int flog_close(int fdout)
 	return 0;
 }
 
-int flog_encode_msg(int fdout, unsigned int nargs, unsigned int mask, const char *format, ...)
+int flog_encode_msg(int loglevel, unsigned int nargs, unsigned int mask, const char *format, ...)
 {
 	flog_msg_t *m;
 	va_list argptr;
 	char *str_start, *p;
 	size_t i;
+	int fdout;
+	unsigned int current_loglevel;
+
+	if (!opts.log_in_binary){
+
+		va_start(argptr, format);
+		vprint_on_level(loglevel, format, argptr);
+		va_end(argptr);
+
```

Comment:
Put "return" here (or forward goto) to avoid indentation shift for the rest of the function.

Reply:
Okay Sir, I will add a goto statement to transfer flow back to the regular printer.